dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java

```diff
@@ -324,6 +333,92 @@ default String getURLFromEntry(String dockerEntry, ToolDAO toolDAO) {
         return url;
     }
 
+    // TODO: Implement for DockerHub, then gitlab and seven bridges;
+    default Set<Image> getImagesFromRegistry(String toolsJSONTable) {
+        ArrayList<Map<String, String>> dockerTools = new ArrayList<>();
+        Gson gson = new Gson();
```

Comment:
Can create once for the class since it doesn't change from method call to method call